nuxeo-distribution/nuxeo-jsf-ui-hotreload-tests/src/test/java/org/nuxeo/ftest/jsf/hotreload/studio/NuxeoITCaseHelper.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * (C) Copyright 2017 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Anahide Tchertchian
+ */
+package org.nuxeo.ftest.jsf.hotreload.studio;
+
+import org.nuxeo.functionaltests.AbstractTest;
+import org.nuxeo.functionaltests.Assert;
+import org.nuxeo.functionaltests.Constants;
+import org.nuxeo.functionaltests.pages.DocumentBasePage;
+import org.nuxeo.functionaltests.pages.forms.WorkspaceCreationFormPage;
+import org.nuxeo.functionaltests.pages.tabs.WorkspacesContentTabSubPage;
+import org.openqa.selenium.By;
+
+/**
+ * Case helper, with all helper methods to share for all tests
+ *
+ * @since 2.9
+ */
+public abstract class NuxeoITCaseHelper extends AbstractTest {
+
+    protected static final String TEST_WS_TITLE = "Studio Test Workspace";
+
+    protected DocumentBasePage getOrCreateTestWorkspace() {
+        DocumentBasePage wsPage = getOrCreateTestWorkspace(TEST_WS_TITLE, "Testing Nuxeo Studio doc type");
+        return wsPage;
+    }
```

Comment:
Why not creating this workspace through REST as we delete it through REST. Note that creating it with `RestHelper` will allow us to remove line `RestHelper.deleteDocument(WORKSPACES_PATH + TEST_WS_TITLE);` and probably this class.